src/main/scala/com/box/castle/core/leader/TaskManager.scala

```diff
@@ -59,20 +66,25 @@ class TaskManager(val leaderConfig: LeaderConfig, val committerConfigs: Iterable
       tasks
   }
 
-  private def matchTopicsRegex(topicsRegex: Regex, kafkaTopic: KafkaTopic, committerId: String, committerIds: Set[String]) = {
-    kafkaTopic.name match {
-      case topicsRegex(_*) => committerIds + committerId
-      case _ => committerIds
-    }
+  protected[leader] def matchTopicsRegex(topicsRegex: Regex, kafkaTopic: KafkaTopic, committerId: String, committerIds: Set[String]) = {
```

Comment:
1. Need a function signature
2. What's the difference between committerId & committerIds?